src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -192,49 +197,66 @@ def list_releases(name, resource_group_name):
     json_request = req.json()
     return json_request
 
+def _gitroot():
+    """
+    Gets the absolute path of the repository root
+    """
+    try:
+        base = check_output(['git', 'rev-parse', '--show-toplevel'])
+    except:
+        raise IOError('Current working directory is not a git repository')
```

Comment:
All exceptions are being caught but exceptions could be raised for different reasons.
For example, the way the code is right now, if I don't have 'git' installed on my machine an exception will be raised and then the code will `raise IOError('Current working directory is not a git repository')` which isn't true.

Just as a suggestion, you could:
1st. catch `OSError` separately which will be thrown if git isn't installed.
2nd. catch `CalledProcessError` which will be thrown if git returned an error code.

https://docs.python.org/2/library/subprocess.html#subprocess.check_output
https://docs.python.org/2/library/subprocess.html#exceptions